tfplan/json_plan.go

```diff
@@ -28,19 +25,13 @@ const maxChildModuleLevel = 10000
 // jsonPlan structure used to parse Terraform 12 plan exported in json format by 'terraform show -json ./binary_plan.tfplan' command.
```

Comment:
L23 (`maxChildModuleLevel`) is now unused, right?

Reply:
yep! removed in ec57ba01